references.bib

```diff
@@ -476,6 +532,7 @@ @article{SkillingJ.1975
 title = {{The Complete Set of Uniform Polyhedra}},
 volume = {278},
 year = {1975},
+url={},
```

Comment:
Why is this url empty?

Reply:
This has been removed.

Reply:
But we need a URL for every BibTeX entry.

Reply:
I have added the URL